plans/integration/changed-yum-conf/vm_centos7.fmf

```diff
@@ -0,0 +1,14 @@
+provision:
+    how: libvirt
+    origin_vm_name: c2r_centos7_template
+    develop: true
+
+/good:
+    discover+:
+        filter+:
+            - 'tag: good_test'
+    prepare:
+        how: ansible
+        playbook:
+            - tests/ansible_collections/main.yml
+            - tests/integration/changed-yum-conf/ansible/change_yum_conf.yml
```

Comment:
```suggestion
    prepare+:
        - name: change yum config
          how: ansible
          playbook:
            - tests/integration/changed-yum-conf/ansible/change_yum_conf.yml
```

Reply:
You need to extend the metadata, rather than overwrite it

Reply:
What about https://github.com/oamg/convert2rhel/pull/189#discussion_r668835784? I.e. merge main.fmf and vm_centos7.fmf into one.

Reply:
yes, sure, we're no longer creating separate fmf for different vms. Agree.
@hosekadam example of merging https://github.com/oamg/convert2rhel/blob/main/plans/integration/releasever-as-mapping/main.fmf